scripts/geopmpy/launcher.py

```diff
@@ -49,13 +49,13 @@ def __init__(self):
 
     def create(self, argv, num_rank=None, num_node=None, cpu_per_rank=None, timeout=None,
                time_limit=None, job_name=None, node_list=None, exclude_list=None, host_file=None,
-               partition=None, reservation=None, quiet=None):
+               bootstrap=None, partition=None, reservation=None, quiet=None):
```

Comment:
Put bootstrap at the end of the argument list to preserve backward compatibility (doubt anyone is calling this with just positional arguments, but it is possible)